plugins/woocommerce/src/Internal/Brands.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Brands class file.
+ */
+
+declare( strict_types = 1);
+
+namespace Automattic\WooCommerce\Internal;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class to initiate Brands functionality in core.
+ */
+class Brands {
+
+	/**
+	 * Class initialization
+	 *
+	 * @internal
+	 */
+	final public static function init() {
+
+		if ( ! self::is_enabled() ) {
+			return;
+		}
+
+		// If the WooCommerce Brands plugin is activated via the WP CLI using the '--skip-plugins' flag, deactivate it here.
+		if ( function_exists( 'wc_brands_init' ) ) {
+			remove_action( 'plugins_loaded', 'wc_brands_init', 1 );
+		}
+
+		include_once WC_ABSPATH . 'includes/class-wc-brands.php';
+		include_once WC_ABSPATH . 'includes/class-wc-brands-coupons.php';
+		include_once WC_ABSPATH . 'includes/class-wc-brands-brand-settings-manager.php';
+		include_once WC_ABSPATH . 'includes/wc-brands-functions.php';
+
+		if ( wc_current_theme_is_fse_theme() ) {
+			include_once WC_ABSPATH . 'includes/blocks/class-wc-brands-block-templates.php';
+			include_once WC_ABSPATH . 'includes/blocks/class-wc-brands-block-template-utils-duplicated.php';
+		}
+
+		if ( is_admin() ) {
+			include_once WC_ABSPATH . 'includes/admin/class-wc-admin-brands.php';
+		}
+	}
+
+	public static function is_enabled() {
+		$assignment = get_option( 'woocommerce_remote_variant_assignment', 0 );
```

Comment:
let's set this to false for the first release, and then we will enable it in a subsequent one, wdyt? also, we should probably pick some random start number which is not `0`, otherwise for some shops, all the experimental features will be enabled at once.

Reply:
Good idea to have the default value to `false` and prevent stores that don't have this option from loading this package. 

Regarding picking a random number: 
- the `is_enabled` function is isolated and doesn't update the `woocommerce_remote_variant_assignment` and;
- the main WooCommerce file is responsible for picking a random number for `woocommerce_remote_variant_assignment` and then updating the option: [here](https://github.com/woocommerce/woocommerce/blob/057e118a3443721d947aa92c0487aac1238d288a/plugins/woocommerce/includes/class-woocommerce.php#L309-L310) and [here](https://github.com/woocommerce/woocommerce/blob/057e118a3443721d947aa92c0487aac1238d288a/plugins/woocommerce/includes/class-woocommerce.php#L370).

That said, I don't think that individual packages should update this option, as this controls many experimental features and not only Brands.